src/main/java/org/broadinstitute/hellbender/cmdline/argumentcollections/GATKReadFilterPluginArgumentCollection.java

```diff
@@ -0,0 +1,23 @@
+package org.broadinstitute.hellbender.cmdline.argumentcollections;
+
+import java.io.Serializable;
+import java.util.Collection;
+
+/**
+ * An abstract ArgumentCollection for specifying parameters for {@link org.broadinstitute.hellbender.cmdline.GATKPlugin.GATKReadFilterPluginDescriptor}.
+ *
+ * @author Daniel Gomez-Sanchez (magicDGS)
+ */
+public abstract class GATKReadFilterPluginArgumentCollection implements Serializable {
+    private static final long serialVersionUID = 1L;
+
+    /** Returns the enabled filter names. */
+    public abstract Collection<String> getEnabledReadFilters();
```

Comment:
This should to by typed as a List to maintain ordering (Collection doesn't necessarily preserve order), and add a comment saying that order should be honored.